### ISSUE TITLE
OAE - Reordenació camps pantalla Serveis

### ISSUE THREAD

**Issue by scalafat-dgtic** [GovernIB/rolsac2#134]
Organització en apartats
- DADES SIA
- DADES DE PUBLICACIÓ
- DADES
- ÒRGANS
- DADES DE CONTACTE
- CANALS DE PRESENTACIÓ
- INFORMACIÓ SOBRE PROTECCIÓ DE DADES PERSONALS

Canvis de comportament:
- Al "Canals de presentació", el check de "Telemàtic" ha de ser el primer i, per defecte, ha d'estar marcat (amb el comportament corresponent)
- Els camps "Responsable del tractament" i "Drets" no han de ser editables
- Els camps "Finalitat" i "Destinari" han de ser editables per procediment (tot i que agafi el valor per defecte des de l'Adm.Entitat) i el valor final a emmagatzemar ha de ser el que modifiqui l'usuari.

Respecte l'ordre, les imatges següents marquen la disposició provisional canviada:

![image](https://github.com/GovernIB/rolsac2/assets/44995350/c5a16128-180e-44cb-81dd-77fa76546228)

![image](https://github.com/GovernIB/rolsac2/assets/44995350/49afe74f-313d-4bcd-8a92-14eda289799d)

**Comment by slromero-indra** [comment]
Actualmente, si no se marca la LOPD como activa, se oculta toda la información de lopd. Hemos observado en la imagen que la legimitación se observa, debe mostrarse aunque no esté activa? Entiendo que debe ser como en la issue de procedimientos https://github.com/GovernIB/rolsac2/issues/133 que ocupa una línea completa (manteniendo lo de no mostrarlo si el check no está activo).

**Comment by scalafat-dgtic** [comment]
Sí, debe mantener el mismo comportamiento (si está activo se muestra; si no, no). Al final estas modificaciones que veis en la imagen las hicimos desde la función de inspeccionar del navegador. Se coló lo del check en algún momento.

**Comment by slromero-indra** [comment]
Se han subido los cambios en el tag rolsac-2.0-S18_1, podéis revisarlo?

**Comment by scalafat-dgtic** [comment]
Como en #133, faltaría ordenar los módulos de la derecha de modo que quede de la siguiente manera, de arriba a abajo:

- PUBLIC OBJECTIU
- ASSIGNACIÓ TEMÀTICA - Matèries
- NORMATIVA DEL SERVEI
- DOCUMENTS RELACIONATS

**Comment by slromero-indra** [comment]
Hemos generado un tag nuevo [rolsac-2.0-S18_2 ](https://github.com/GovernIB/rolsac2/tree/rolsac-2.0-S18_2) que reorganiza los módulos laterales, cuando podáis lo validáis
